docker/Dockerfile

```diff
@@ -86,6 +86,12 @@ COPY $EXTERNAL_SOURCE_NESTED/app-config*.yaml ./
 # hadolint ignore=DL3059
 RUN $YARN install --frozen-lockfile --production --network-timeout 600000
 
+# Stage 4.5 - build dynamic plugins
+FROM skeleton AS dynamic-plugins
```

Comment:
do we realllllly need yet another named layer? this is just another thing I have to remove downstream. What's the value in having all these names layers? Could we just have a `build` and a `runner` layer?

Reply:
No idea :-)
I just tried to respect the way it was done, but have no idea why the structures between Dockerfile and brew.Dockerfile need to be different.

Reply:
brew/osbs's implementation can't use named layers that refer to previous named layers. It can do multiple

```
FROM registry/org/container:tag as label
```

but not 

```
FROM previousLabel as newLabel
```

So downstream I ripped out all the intermediate layers that @schultzp2020 added for [reasons] and we have a working build again. If you want to follow his style, it just means more things to remove downstream in brew.Dockerfile.